docs/querying/api.md

```diff
@@ -653,6 +653,52 @@ curl -G http://localhost:9091/api/v1/targets/metadata \
 }
 ```
 
+## Querying metric metadata
+
+Similarly to the previous endpoint `api/v1/targets/metadata`, it returns metadata about metrics currently scrapped by targets. However, it does not provide any target information.
+This is considered **experimental** and might change in the future.
+
+```
+GET /api/v1/metadata
+```
+
+URL query parameters:
+
+- `limit=<number>`: Maximum number of metrics to return.
+
+The `data` section of the query result consists of an object where the keys represent a metric name and the values are a list of the non-matching metadata found on the scraped targets. The metadata that is equal across all targets is collapsed, and aggregates under the same metric name the ones that differ.
```

Comment:
This is a bit verbose, but also confusing. How about the following:

"The `data` section of the query result consists of an object where each key is a metric name and each value is a list of unique metadata objects, as exposed for that metric name across all targets."